infer/environment.cc

```diff
@@ -710,18 +710,13 @@ void Environment::updateKnowledge(core::Context ctx, cfg::LocalRef local, core::
 
         const auto &argSymData = argSym.data(ctx);
         if (argSymData->isModule()) {
-            if (!recvType.isUntyped() && recvType.derivesFrom(ctx, core::Symbols::Class())) {
```

Comment:
What's the intuition for why we no longer test that the receiver is not `T.untyped`?

Reply:
That clears it up, thanks!